src/components/InputField/index.js

```diff
@@ -15,7 +15,7 @@ const InputField = ({ meta, children, hint, input }) => (
     <LabelText>{children}</LabelText>
     {hint && <HintText>{hint}</HintText>}
     {meta.touched && meta.error && <ErrorText>{meta.error}</ErrorText>}
-    <Input type="text" {...input} />
+    <Input {...input} />
```

Comment:
why get rid of the type?

Reply:
i thought its not needed since we are setting a defaultProp of type='text'?

Reply:
oh are we, ok never mind